### DISCUSSION TITLE
Moving Lights setup options

### DISCUSSION THREAD

**Discussion by DizziDeCazz** [spensbot/captivate#28] [Ideas]
Hi. I finally got a chance to check v1.0 and it works well with my setup. It's great and I have fun times a head I'm sure. Fantastic work.
My first issue is the direction of my moving lights. I think you need to add configuration options, especially considering that many users won't be using captivate only so the so rearranging the physical setup to suit captivate is not practical. Mine currently point the wrong way.
- Set a default (home) pan and tilt position, usually 128 - the middle point.
- Invert pan and/or tilt.
I think these could be worked around by modifying the fixture profile but would be nicer if they had config options.

Also I can't seem to get the DMX Out view to show anything above ch 255.

**Comment by arlandlove** [comment]
Having the same issue with the DMX output only showing up to channel 255.
